modules/qna/src/views/full/index.jsx

```diff
@@ -418,11 +428,18 @@ export default class QnaAdmin extends Component {
   deleteItem = id => () => {
     const needDelete = confirm('Do you want to delete the question?')
     const { filterQuestion, filterCategory, page } = this.state
+    let deleteContentElements = false
+    if (this.state.items.filter(item => item.id == id)[0].data.answers.filter(a => a.contentId != undefined).length > 0) {
```

Comment:
There's a lot of stuff happening here, can you extract some of it & it looks to be missing null checks (& strict checks)